apps/etterlatte-saksbehandling-ui/client/src/components/aktivitetsplikt/AvbrytAktivitetspliktOppgave.tsx

```diff
@@ -0,0 +1,37 @@
+import { useAktivitetspliktOppgaveVurdering } from '~components/aktivitetsplikt/AktivitetspliktOppgaveVurderingRoutes'
+import { useApiCall } from '~shared/hooks/useApiCall'
+import { avbrytOppgaveMedMerknad } from '~shared/api/oppgaver'
+import { Button } from '@navikt/ds-react'
+import { isPending } from '~shared/api/apiUtils'
+import React from 'react'
+import { erOppgaveRedigerbar } from '~shared/types/oppgave'
+import { useInnloggetSaksbehandler } from '~components/behandling/useInnloggetSaksbehandler'
+import { useForm } from 'react-hook-form'
+
+export function AvbrytAktivitetspliktOppgave() {
+  const innloggetSaksbehandler = useInnloggetSaksbehandler()
+  const { oppgave } = useAktivitetspliktOppgaveVurdering()
+
+  const erTildeltSaksbehandler = innloggetSaksbehandler.ident === oppgave.saksbehandler?.ident
+  const kanRedigeres = erOppgaveRedigerbar(oppgave.status)
+
+  const [avbrytOppgaveStatus, avbrytOppgave] = useApiCall(avbrytOppgaveMedMerknad)
+
+  const { handleSubmit } = useForm<{ kommentar: string }>({ defaultValues: { kommentar: '' } })
```

Comment:
pirk pirk, men i objektet som sendes til backend så brukes `merknad`. Ikke `kommentar`

Reply:
Eller kommentar er vel det som blir brukt i skjemaet, så her blir det mer riktig

Reply:
Det er en copy-paste feil egentlig.. har ikke lagt til rette for at saksbehandler selv kan kommentere merknad. Burde vi?

Reply:
Ordner det, men merkand burde være påkrevd for å sikre nok informasjon om hvorfor oppgaven ble avbrutt av saksbehandler.